### DISCUSSION TITLE
I made a copy in Nuxt.js and Vuetify

### DISCUSSION THREAD

**Discussion by MehdiRtal** [arifszn/gitprofile#134] [Show and tell]
Here's the repo : [Gitfolio](https://github.com/MehdiRtal/Gitfolio)
It's still WIP, I'll probably add some additional features such as skeleton loadings and stuff.
I also hope it's ok copying a part from your README, I'll change it later.

**Comment by arifszn** [comment]
Awesome.
Can you please replace the credit section with GitProfile's link instead of my name? Thanks.

**Reply by MehdiRtal** [reply]
Done!
